classes/07class/exercises/c07-serverless01/alanlima/terraform/c07-serverless01.tf

```diff
@@ -0,0 +1,115 @@
+data "archive_file" "photos_lambda" {
```

Comment:
Nice zipping :) 
Quick tip, usually logic of what is zipped go in a make target `package`.